tcms/testplans/templates/testplans/get.html

```diff
@@ -324,6 +316,14 @@ <h2 class="card-pf-title">
     </div>
 </template>
 
+<template id="toolbar-dropdown">
+    <li class="selected" data-filter-type="summary"><a>{% trans 'Summary' %}</a></li>
+    <li data-filter-type="category"><a>{% trans 'Category' %}</a></li>
+    <li data-filter-type="priority"><a>{% trans 'Priority' %}</a></li>
+    <li data-filter-type="author"><a>{% trans 'Author' %}</a></li>
+    <li data-filter-type="default_tester"><a>{% trans 'Default tester' %}</a></li>
+</template>
```

Comment:
FTR I meant an HTML file which you can `include` directly in this template but works like this for now. Will merge as-is.